http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -56,6 +60,9 @@
     protected final String name;
     protected final Class<? extends Annotation> annotation;
     protected Binder binder;
+    // we have a single instance of the JettySharedIoPoolModule to make sure
```

Comment:
For multiline comments like this, it's easier to read as a sentence (so capitalize and add period). Also add a newline before to separate it from the other fields. And move above, since we normally put constants above fields.